Formula/vice.rb

```diff
@@ -14,59 +13,65 @@ class Vice < Formula
     sha256 "ab4044f958907bd7d756575fc97e0e42ffc24307c621176da0d0522feadb22f4" => :yosemite
   end
 
+  head do
+    url "https://svn.code.sf.net/p/vice-emu/code/trunk/vice"
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "pkg-config" => :build
   depends_on "texinfo" => :build
   depends_on "yasm" => :build
-  depends_on "ffmpeg"
+  depends_on "xa" => :build
+  depends_on "ffmpeg@2.8"
   depends_on "flac"
   depends_on "giflib"
   depends_on "jpeg"
   depends_on "lame"
   depends_on "libogg"
   depends_on "libpng"
   depends_on "libvorbis"
+  depends_on "mpg123"
   depends_on "portaudio"
-  depends_on "sdl2"
   depends_on "xz"
-
-  # needed to avoid Makefile errors with the vendored ffmpeg 2.4.2
-  resource "make" do
-    url "https://ftp.gnu.org/gnu/make/make-4.2.1.tar.bz2"
-    mirror "https://ftpmirror.gnu.org/make/make-4.2.1.tar.bz2"
-    sha256 "d6e262bf3601b42d2b1e4ef8310029e1dcf20083c5446b4b7aa67081fdffc589"
-  end
+  depends_on "sdl2"
+  depends_on "gtk+3" => :optional
 
   def install
-    resource("make").stage do
-      system "./configure", "--prefix=#{buildpath}/vendor/make",
-                            "--disable-dependency-tracking"
-      system "make", "install"
+    ENV.prepend_path "PATH", "/usr/local/opt/ffmpeg@2.8/bin"
+    ENV.prepend_path "PKG_CONFIG_PATH", "/usr/local/opt/ffmpeg@2.8/lib/pkgconfig"
```

Comment:
These paths should't be hardcoded, otherwise any prefix besides `/usr/local/` would break. What does work is to use `Formula["foobar"].opt_bin` and `Formula["foobar"].opt_lib/"pkgconfig"` instead.

Reply:
Forgot that one…